cmd/pj-rehearse/main.go

```diff
@@ -339,6 +331,24 @@ func rehearseMain() error {
 		return fmt.Errorf(jobValidationOutput)
 	}
 
+	cleanup, err := setupDependencies(
+		presubmitsToRehearse,
+		prNumber,
+		buildClusterConfigs,
+		logger,
+		prConfig.Prow.ProwJobNamespace,
+		prConfig.Prow.PodNamespace,
+		pluginConfig,
+		o.releaseRepoPath,
+		o.dryRun,
+		changedTemplates,
+		changedClusterProfiles)
```

Comment:
eww. I don't see an apparent way how to make it simpler, so I guess we'll have to run with it.